### PR TITLE
[CMake] Always build the "non-overlay" apinotes

### PR DESCRIPTION
**Pull request by Rostepher**
# Purpose

This PR forces swiftlang to always "rebuild" the "non-overlay" apinotes found in the `apinotes` directory when compiling.

rdar://40496966

### CODE REVIEW

**Comment by Rostepher** [comment]
@swift-ci please test

**Comment by jrose-apple** [comment]
I think this will build them *both* ways; it'd really be better to stop building them with the overlays. Or did you want to save that for a follow-up?

**Comment by Rostepher** [comment]
Closing this, I don't think this is the right solution.

**Comment by Rostepher** [comment]
@swift-ci test

File: cmake/modules/AddSwift.cmake [hunk 1]
```diff
@@ -719,9 +719,9 @@ function(_add_swift_library_single target name)
           ${SWIFTLIB_SINGLE_SDK} STREQUAL "OSX")
         # HACK: don't build WatchKit API notes for OS X.
       else()
-        if (NOT IS_DIRECTORY "${SWIFT_SOURCE_DIR}/stdlib/public/SDK/${framework_name}")
-          list(APPEND SWIFTLIB_SINGLE_API_NOTES "${framework_name}")
-        endif()
+        # Always build the "non-overlay" apinotes to keep them in sync
```
Comment (jrose-apple) [inline cmake/modules/AddSwift.cmake]: Comment nitpick: this is saying "Always build *all* API notes", not just the non-overlay ones.

**Review by jrose-apple — APPROVED** [review]

File: cmake/modules/AddSwift.cmake (hunk 1 above)
Comment (Rostepher) [reply cmake/modules/AddSwift.cmake]: Well, it's building all the apinotes found in `swift/apinotes/`, but I can adjust the comment.

**Pull request merged**